### ISSUE TITLE
[BUG] Doesn't update through BRAT

### ISSUE THREAD

**Issue by huachuman** [yan42685/obsidian-clever-search#5]
Hi there, I ran into an issue where BRAT refuses/fails to update the plugin.

The BRAT developer has said the following:

> The problem with the example repo is that while it does have manifest in the root, it doesn't have a release in the repo corresponding to the release version in manifest.json.
> 
> BRAT first reads manifest.json to get the release version, then downloads that release.
> 
> This is a mistake many devs make when starting Obsidian development. Obsidian also can't download the plugin unless there is a corresponding release.
> 
> This is one of the values of BRAT; in effect, BRAT is showing the repo is not configured correctly.

https://github.com/TfTHacker/obsidian42-brat/issues/75

**Comment by yan42685** [comment]
Thanks for the heads-up! I was prepping for the release of version 0.1 and updated the manifest in advance, but didn't create the release yet. I wasn't aware of this issue before. I'll make sure to release the 0.1 version of the plugin promptly, which supports in-vault search.

Before the release, I still have a few issues to resolve, one of which is to confirm that users not living in China can access the resource files I've uploaded to a Chinese website. If you have time, please try clicking this online txt link to see if it opens: https://gitee.com/alexyan333777/obsidian-clever-search/raw/dev/assets/for-program/stop-words-en.txt. Since many Chinese users cannot access GitHub and npm, I had to upload the resource files to a Chinese website instead.

**Comment by yan42685** [comment]
I have now downgraded the version in manifest.json to 0.0.4. Can you update normally now?

**Comment by huachuman** [comment]
> Before the release, I still have a few issues to resolve, one of which is to confirm that users not living in China can access the resource files I've uploaded to a Chinese website. If you have time, please try clicking this online txt link to see if it opens: https://gitee.com/alexyan333777/obsidian-clever-search/raw/dev/assets/for-program/stop-words-en.txt. Since many Chinese users cannot access GitHub and npm, I had to upload the resource files to a Chinese website instead.

[session-ff417fbf] Route error: ["Access denied", 4]

That's fascinating I had no idea a Canadian IP would be blocked from accessing a chinese website.

**Comment by yan42685** [comment]
Indeed, it appears that IP-based access restrictions aren't exclusive to any particular region. Chinese IPs also generally face difficulties accessing websites from many other countries. However, no worries! After some trial, I've found that Chinese IPs can successfully access `bitbucket.org`, which is available for most of the users. So this problem has been resolved. Anyway, thank you for your assistance in testing. It has broadened my understanding of the network conditions in China.
